rotkehlchen/db/schema.py

```diff
@@ -625,6 +625,9 @@
 """  # noqa: E501
 
 
+# usd value is a column of the table because we sort by value in the fiat currency and that price
```

Comment:
is it value or price? Please adjust either the docstring or the name of the column accordingly